suspense/futures.py

```diff
@@ -1,20 +1,13 @@
-import concurrent.futures
 import uuid
 
-__futures = {}
+from asgiref.sync import sync_to_async
 
 
-def create(nodelist, context):
-    key = str(uuid.uuid4())
-
-    executor = concurrent.futures.ThreadPoolExecutor()
-    __futures[key] = executor.submit(nodelist.render, context)
+def task(key, nodelist, context):
+    return key, nodelist.render(context)
 
-    return key
 
+def create(nodelist, context):
+    key = str(uuid.uuid4())
 
-def get(key):
-    if key not in __futures:
-        return None
-
-    return __futures.pop(key).result()
+    return key, sync_to_async(task)(key, nodelist, context)
```

Comment:
I just realized something fundamentally wrong with what I did. There is no paralleliem/concurrency at all. 

Each suspense blocks will run synchronously one after the other.

I have a solution in mind I will experiment.

Reply:
Your latest change breaks suspense. Now we get the full page instead of getting the fallback first.

Reply:
with `runserver_plus` it seems to work but not on asgi. The latest change is no longer asgi since nothing was async. everything was running synchronously on the same thread.

Reply:
works also corretly on wsgi:
```shell
 PYTHONPATH=$PWD/../ python -m gunicorn example.wsgi:application
```

Reply:
Indeed. For a WSGI server, it works. I apologize.